weave

```diff
@@ -554,6 +554,11 @@ echo_addresses() {
     echo $1 $2 $3
 }
 
+peer_args() {
+  local joined=$(printf "&peer=%s" "${@}")
+  echo ${joined:1}
+}
```

Comment:
```
peer_args() {
  res=''
  sep=''
  for p in "$@" ; do
      res="$res${sep}peer=$p"
      sep="&"
  done
  echo "$res"
}
```

(Note that `local` is not in POSIX, even though dash supports it. I don't know whether we care about that.)